portfolio.html

```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+
+            <div class="card-work-1">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder5">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+
+            <div class="card-work-1">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder6">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+
+            <div class="card-work-1">
+                <img src="./images/mobile_version/Card/Img_Plaholder.png" class="image" alt="image placeholder7">
+                
+                <div class="content">
+                <h1 class="header-work-1">Multi-Post Stories <br> Gain+Glory</h1>
+                <ul class="list-languages">
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="css">css</li>
+                    <li class="javascript">JavScript</li>
+                    <li class="html">html</li>
+                </ul>
+                <div class="button-frame">
+                    <button type="button" class="btn-green">See Project</button>
+                </div> 
+                </div>
+            </div>   
+        </div>
+    </section>
+
+    <section class="about-me">
+        <div class="text-about-me">
+            <h1 class="about">About me</h1>
```

Comment:
- [ ] Kindly make the header tag any hx tag from h2 - h6,  this will be more semantic for your website  👍🏿